src/wallet/wallet.cpp

```diff
@@ -3673,6 +3668,33 @@ bool CMerkleTx::AcceptToMemoryPool(bool fLimitFree, bool fRejectAbsurdFee)
     return ::AcceptToMemoryPool(mempool, state, *this, fLimitFree, NULL, fRejectAbsurdFee);
 }
 
+void CWallet::GetFilteredTransactions(std::multimap<int64_t, CWalletTx >& outEntries, const std::string& address)
+{
+    LOCK2(cs_main, cs_wallet);
+    CScript scriptPubKey;
+
+    if(address.compare("*")!=0){
```

Comment:
Elsewhere (in rpcwallet calling this method) the comparison operator (!=) is used for string, it would be best to use always the same notation.